README.md

```diff
@@ -42,7 +42,7 @@ docker build -t docbot .
 Then run it like this:
 
 ```sh
-docker run -d -p5000:5000 -e GITHUB_TOKEN=<token> --name docbot docbot
+docker run -d -p5000:5000 -e GITHUB_TOKEN=<token> -e GITHUB_SECRET_TOKEN=<secret_token> --name docbot docbot
```

Comment:
What do you think about `GITHUB_SECRET_TOKEN` -> `GITHUB_SIGN_TOKEN`?

Reply:
Thanks! Fixed.